benchmarks/src/it/scala/akka/kafka/benchmarks/Producer.scala

```diff
@@ -8,23 +8,23 @@ class ApacheKafkaPlainProducer extends BenchmarksBase() {
   private val prefix = "apache-kafka-plain-producer"
 
   it should "bench with small messages" in {
-    val cmd = RunTestCommand(prefix, bootstrapServers, topic_2000_100)
+    val cmd = RunTestCommand(prefix, bootstrapServers, topic_2000_100.freshTopic)
     runPerfTest(cmd, KafkaProducerFixtures.initializedProducer(cmd), KafkaProducerBenchmarks.plainFlow)
   }
 
   it should "bench with 500b messages" in {
-    val cmd = RunTestCommand(prefix + "-500b", bootstrapServers, topic_2000_500)
+    val cmd = RunTestCommand(prefix + "-500b", bootstrapServers, topic_2000_500.freshTopic)
     runPerfTest(cmd, KafkaProducerFixtures.initializedProducer(cmd), KafkaProducerBenchmarks.plainFlow)
   }
 
   it should "bench with normal messages" in {
-    val cmd = RunTestCommand(prefix + "-normal-msg", bootstrapServers, topic_2000_5000)
+    val cmd = RunTestCommand(prefix + "-normal-msg", bootstrapServers, topic_2000_5000.freshTopic)
     runPerfTest(cmd, KafkaProducerFixtures.initializedProducer(cmd), KafkaProducerBenchmarks.plainFlow)
   }
 
   it should "bench with normal messages written to 8 partitions" in {
     val cmd =
-      RunTestCommand(prefix + "-normal-msg-8-partitions", bootstrapServers, topic_2000_5000_8)
+      RunTestCommand(prefix + "-normal-msg-8-partitions", bootstrapServers, topic_2000_5000_8.freshTopic)
```

Comment:
I don't see what changed here between #902 and here.  If it was reusing the same topic, it was doing that before as well.  The new benchmark for 500b messages would have had its own random topic id.